client/components/account-balances/test/index.test.tsx

```diff
@@ -438,4 +450,34 @@ describe( 'AccountBalancesTabPanel', () => {
 			documentationUrls.depositSchedule
 		);
 	} );
+
+	test( 'renders instant deposit button correctly', () => {
+		mockOverviews( [ createMockOverview( 'usd', 10000, 20000, 30000 ) ] );
+		render( <AccountBalancesTabPanel /> );
+
+		const instantDepositButton = screen.getByRole( 'button', {
+			name: 'Deposit available funds',
+		} );
+		expect( instantDepositButton ).toBeInTheDocument();
```

Comment:
Nit: since we're using a `get` react-resting-library query, we don't need to also `expect()`, since `getByRole()` represents the same assertion as `expect( instantDepositButton ).toBeInTheDocument()` ([docs ref](https://testing-library.com/docs/queries/about#types-of-queries)).
```suggestion
		screen.getByRole( 'button', {
			name: 'Deposit available funds',
		} );
```

Reply:
Nice! Suggestion committed.